### PR TITLE
[TITAN-337] - Shopify Blueprint Documentation

### PR DESCRIPTION
**Pull request by RossoMaguire**
# Description

This change adds the documentation for the Shopify Blueprint deployment and configuration via Atlas Dashboard. 

We also need a tile added to the home page https://wpengine.com/builders/headless/

## Scope of change

Please check the option that describes the scope of the change you are requesting.

- [ ] Small (fixing typos, changing lines of an existing doc, etc.)
- [ ] Medium (changing larger sections of an existing doc that could require back-and-forth)
- [x] Large (adding new pages or making changes to multiple pages)

### CODE REVIEW

File: atlas-shopify-blueprint/configuring-shopify.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+slug: /atlas-shopify-blueprint/configuring-shopify
+title: Atlas Shopify Blueprint - Configuring Shopify
+description: Configure the Shopify Blueprint with Shopify Storefront API credentials.
+---
+
+## Configuring Shopify
+
+In order to list products from your **Shopify** store in the Blueprint once it has been deployed, you need to add your Shopify Access Token and Shopify GraphQL URL as Environment Variables.
```
Comment (alvarocavalcanti) [inline atlas-shopify-blueprint/configuring-shopify.mdx:9]: We can at least get this bit more aligned with Shopify's terminology, ahead of [TITAN-385](https://wpengine.atlassian.net/browse/TITAN-385):

```suggestion
In order to list products from your **Shopify** store in the Blueprint once it has been deployed, you need to add your [Shopify Headless Public Access Token](https://shopify.dev/docs/api/usage/authentication#getting-started-with-public-access) and [Shopify GraphQL URL](https://shopify.dev/docs/api/storefront#endpoints) as Environment Variables.
```

File: atlas-shopify-blueprint/configuring-shopify.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+The Environment Variables that need to be set in addition to the default ones should be named as follows:
+
+- NEXT_PUBLIC_SHOPIFY_TOKEN
+- NEXT_PUBLIC_SHOPIFY_URL
```
Comment (alvarocavalcanti) [inline atlas-shopify-blueprint/configuring-shopify.mdx:14]:
```suggestion
- `NEXT_PUBLIC_SHOPIFY_TOKEN`: See [Getting started with public access](https://shopify.dev/docs/api/usage/authentication#getting-started-with-public-access)
- `NEXT_PUBLIC_SHOPIFY_URL`: See [Storefront API Endpoints](https://shopify.dev/docs/api/storefront#endpoints)
```

File: atlas-shopify-blueprint/configuring-shopify.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The following guide walks through the process of creating Environment Variables in the **Atlas** Dashboard as well as for local development:
+
+[Configuring Atlas Environment Variables &#8594;](https://developers.wpengine.com/docs/atlas/local-app-development/frontend-app-config#2--define-environment-variables)
+
+Once the above **Environment Variables** have been added to the Blueprint, you should manually rebuild the site.
+
+![Rebuild](./images/rebuild.png)
+
+Once completed this should show your products listed on various pages on the frontend.
```
Comment (alvarocavalcanti) [inline atlas-shopify-blueprint/configuring-shopify.mdx:26]: When testing on my install, it took a while and I had to clear the cache. Was that the case with you as well? Should we include this bit or can we consider it that the developer should be aware of this type of behaviour?

**Review by alvarocavalcanti — CHANGES_REQUESTED** [review]

File: atlas-shopify-blueprint/configuring-shopify.mdx (hunk 3 above)
Comment (RossoMaguire) [reply atlas-shopify-blueprint/configuring-shopify.mdx:26]: After a rebuild? Didn't happen for me but I can add a note here as a troubleshooting tip

**Review by alvarocavalcanti — DISMISSED** [review]

**Review by alvarocavalcanti — DISMISSED** [review]
High Noon

File: atlas-shopify-blueprint/introduction.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Once the Blueprint has been built and deployed you will see the following message on the frontend in places where product data should be listed:
+
+![Configuration Message](./images/config-message.png)
+
+## Next Step
+
+[Configure Shopify API via Atlas Environment Variables &#8594;](../atlas-shopify-blueprint/configuring-shopify.mdx)
+
+- Check out the demo of [Atlas Shopify Blueprint](https://atlasshopify.wpengine.com/)
+- Try out a [sandbox version of Atlas](https://wpengine.com/atlas/#atlasplans) where you can install Atlas Commerce Blueprint
```
Comment (graceerixon) [inline atlas-shopify-blueprint/introduction.mdx:29]: Should this say Atlas Shopify Blueprint or is Atlas Commerce Blueprint correct?

Comment (RossoMaguire) [reply atlas-shopify-blueprint/introduction.mdx]: Well spotted! Updated now

**Review by graceerixon — APPROVED** [review]

**Pull request merged**